ketcher-autotests/tests/Reagents/reagents-molecule-above-arrow.spec.ts

```diff
@@ -69,4 +66,49 @@ test.describe('Reagents molecule above arrow', () => {
     await page.getByRole('button', { name: 'MDL Rxnfile V2000' }).click();
     await page.getByRole('option', { name: 'PNG Image' }).click();
   });
+
+  test('Detection text as reagent and render reagent information in PNG format in "Preview" tab', async ({
+    page,
+  }) => {
+    /*
+      Test case: EPMLSOPKET-4697
+      Description: File is shown in the preview with the NH3 text reagent above the arrow and HBr below.
+    */
+    await openFileAndAddToCanvas(
+      'KET/text-reagents-below-and-above-arrow.ket',
+      page,
+    );
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await page.getByRole('button', { name: 'MDL Rxnfile V2000' }).click();
+    await page.getByRole('option', { name: 'PNG Image' }).click();
```

Comment:
I believe we can create a function for that with parameter for 'PNG Image'/'SVG Document'/e.t.c